packages/profiles/source/signed-transaction.dto.ts

```diff
@@ -59,7 +59,24 @@ export class ExtendedSignedTransactionData {
 	}
 
 	public isReturn(): boolean {
-		return this.isSent() && this.isReceived();
+		if (this.isTransfer()) {
+			return this.isSent() && this.isReceived();
+		}
+
+		if (this.isMultiPayment()) {
+			let isReturn = true;
+
+			for (const recipient of this.recipients().values()) {
+				if (recipient.address !== this.sender()) {
+					isReturn = false;
+					break;
+				}
+			}
+
+			return isReturn;
+		}
+
```

Comment:
The same as above but for the signed transactions.

Reply:
I tried that before but couldn't because I couldn't find a way to access given wallet in coin class.
https://github.com/ArdentHQ/platform-sdk/blob/d5b83462bf14c6ae1eb4556ffdfbf1e155999cb4/packages/profiles/source/signed-transaction.dto.ts#L83